docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,239 @@
+
+For connectors REST API, we suggest using URI versioning as the most commonly used and easiest to check.
+Which means, all REST requests to for example, version 3 will have URI prefix `/v3`. Currently, Fybrik connectors do not
+have the versioning prefix. We can assume that requests without the prefix relate to the v1 version. 
+
+*Notes:* 
+- The current implementation of controllers REST API does not support any versioning. The new incompatible versions will 
+start from the "/v2" prefix, and we will assume that missing version information is the current (old) version.
+- We do not expect significant API changes. Most of the possible changes are covered by the Taxonomy
+- OpenApi [specification](https://swagger.io/specification/) includes `version` entry in the `Info`, however, the object 
+> "provides metadata about the API. The metadata MAY be used by the clients if needed, and MAY be presented in editing or 
+documentation generation tools for convenience." 
+
+We will update the OpenAPI version values, but they cannot be used for API communications.
+
+## Default Taxonomy
+Fybrik widely uses [Taxonomy](https://fybrik.io/v1.1/concepts/taxonomy/) to defines the terms and related values that 
+need to be commonly understood and supported across the components in the system. Taxonomy provides a mechanism for all 
+Fybrik components to interact using a common dialect. Taxonomy defines a set of immutable structural JSON schemas, or 
+"taxonomies" for resources deployed in Fybrik. However, since the taxonomy is meant to be configurable, a taxonomy.json 
+file is referenced from these schemas for any definition that is customizable. The taxonomy.json file is generated from 
+a base taxonomy and zero or more taxonomy layers:
+- The base taxonomy is maintained by the project and includes all the structural definitions that are subject to 
+customization (e.g.: tags, actions).
+- The taxonomy layers are maintained by users and external systems that add customizations over the base taxonomy 
+(e.g., defining specific tags, actions), therefore the layer are beyond Fybrik responsibility. 
+
+The structural JSON schemas are autogenerated files in the 
+[taxonomy](https://github.com/fybrik/fybrik/tree/master/charts/fybrik/files/taxonomy) direction.
+- The `fybrik_application.json` and `fybrik_module.json` files are generated from corresponded CRDs, so their 
+compatability is covered by the CRDs compatibility.
+- The `datacatalog.json`, `infraatributes.json` and `policymanager.json` files are generated form files in the 
+[pkg/model](https://github.com/fybrik/fybrik/tree/master/pkg/model), therefore any changes in the directory should be  
+validated for backward compatibility.
+- The basic [taxonomy file](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/taxonomy.json), is
+autogenerated from the both above sources. However, users are able to overwrite it, as describe by [Using a Custom 
```

Comment:
described in

Reply:
"describe in" and "describe by" are both correct. ("describe in" is more popular)
https://textranch.com/53597/described-in/or/described-by/

Reply:
as described and not as describe

Reply:
OK